### PR TITLE
Fix docstrings

### PR DESCRIPTION
**Pull request by PedanticHacker**
Fixed some docstrings.

### CODE REVIEW

File: chess/__init__.py [hunk 1]
```diff
@@ -3660,25 +3660,25 @@ def __len__(self) -> int:
     # MutableSet
 
     def add(self, square: Square) -> None:
-        """Adds a square to the set."""
+        """Adds a square to a set."""
```
Comment (niklasf) [inline chess/__init__.py:3663]: This refers to *the* self object, not just any.

Comment (PedanticHacker) [reply chess/__init__.py:3663]: Ah, I see. So don't apply this particular change.

File: chess/__init__.py [hunk 2]
```diff
@@ -3664,21 +3664,21 @@ def add(self, square: Square) -> None:
         self.mask |= BB_SQUARES[square]
 
     def discard(self, square: Square) -> None:
-        """Discards a square from the set."""
+        """Discards a square from a set."""
```
Comment (niklasf) [inline chess/__init__.py:3667]: Same here (and maybe more).

Comment (PedanticHacker) [reply chess/__init__.py]: Everything is fixed in the 3rd commit.

**Pull request closed without merging**